contributors/guide/README.md

```diff
@@ -157,11 +155,7 @@ For questions and troubleshooting, please feel free to use any of the methods of
 
 ## GitHub workflow
 
-To check out code to work on, please refer to [this guide](/contributors/devel/development.md#workflow).
-
-_Improvements needed_
-
-* move github workflow into its own file in this folder.
+To check out code to work on, please refer to [this guide](/github-workflow.md).
```

Comment:
The link to github-workflow is incorrect and different from the link in develoment.md. Development.md file has it at /contributors/guide/github-workflow.md which seems to work.